specifyweb/frontend/js_src/lib/components/Molecules/FilePicker.tsx

```diff
@@ -13,20 +13,22 @@ export function FilePicker({
   id,
   name,
   showFileNames = true,
+  spanClassName = 'h-44 w-full',
   ...rest
 }: Pick<TagProps<'input'>, 'disabled'> & {
   readonly acceptedFormats: RA<string> | undefined;
   // Whether to automatically click on the file input as soon as rendered
   readonly id?: string;
   readonly name?: string;
   readonly showFileNames?: boolean;
+  readonly spanClassName?: string;
 } & (
     | { readonly onFileSelected: (file: File) => void }
     | { readonly onFilesSelected: (files: FileList) => void }
   )): JSX.Element {
   const allowMultiple = 'onFilesSelected' in rest;
   const filePickerButton = React.useRef<HTMLButtonElement>(null);
-
+  const isDisabled = rest.disabled;
```

Comment:
when I left a comment telling you to destructure this prop, I meant adding this after line 16:

```ts
    disabled,
```

or does typescript not allow it?

Reply:
Ah, apologies I misunderstood you the first time. It allowed destructuring at the top!